collector/pg_stat_database.go

```diff
@@ -375,6 +393,11 @@ func (c *PGStatDatabaseCollector) Update(ctx context.Context, instance *instance
 			continue
 		}
 
+		if idleInTransactionTimeAvail && !idleInTransactionTime.Valid {
+			level.Debug(c.log).Log("msg", "Skipping collecting metric because it has no idle_in_transaction_time")
+			continue
```

Comment:
I don't think this needs to skip the whole row in this case. I think this logic can be to skip sending the metric to the channel down on line 540 instead. I know the logic above is this way, but I think technically this addition could break things for other users if this field is NULL, so I want to avoid that.